creator-node/src/AsyncProcessingQueue.js

```diff
@@ -53,8 +54,32 @@ class AsyncProcessingQueue {
 
     this.libs = libs
 
+    const untracedProcessTask = this.processTask
     this.queue.process(MAX_CONCURRENCY, async (job, done) => {
-      await this.processTask(job, done)
+      const { logContext, parentSpanContext, task } = job.data
+      const processTask = instrumentTracing({
```

Comment:
So we have to instrument tracing for every job essentially? Can we not do in the wrapper for process one time?

Reply:
What do you mean? we are instrumenting the process function and getting all the transcoding tasks

Reply:
does this run instrumentTracing on every new job in the queue? cause it's instrumented inside the this.queue.process function

Reply:
Oh.... ya I suppose it does. Would that be an issue?

Reply:
We have to do this because we need that `parentSpanContext`

Reply:
my only concern here is the cost of this call on every upload. like how much latency/resource use does this init take?

Reply:
can we benchmark how long this instrument call takes?

Reply:
<img width="811" alt="Screen Shot 2022-09-02 at 15 28 51" src="https://user-images.githubusercontent.com/17736112/188223903-02da7cec-08ff-405c-ba49-a430172f1dd2.png">

It causes almost nothing to wrap a function. The instrument isn't actually _running_ any code but just returning a new function (and I think getting low level it's JIT compiling the instrumentTracing the first time if it isn't already)

Reply:
you're right, this isn't running any async or intensive code. i think this should be fine for now